multi_import/helpers/files.py

```diff
@@ -15,9 +15,31 @@ def find_format(file_formats, file_format=None):
     )
 
 
-def read(file_formats, file):
+def read_file(file):
     file.seek(0)
-    file_contents = file.read()
+    return file.read()
+
+
+def decode_file(file):
```

Comment:
TODO: make it simpler by using for loop maybe. 
ex: encodings = ['utf-8', 'utf-16', 'latin-1']